### ISSUE TITLE
Convert New themes (colors) to main-theme (CSS only)

### ISSUE THREAD

**Issue by Nainish-Rai** [Nainish-Rai/aesthetic-startpage#41]
you have to convert themes in new-themes folder to new format;
1) index.html will be same for each theme
2) change css to new one given in template-theme normal
3) change var color in each themes for new-theme

**Comment by getmeapencil** [comment]
Do you mean the index.html for new-themes should not be changed?

**Comment by Nainish-Rai** [comment]
@getmeapencil They need to be changed with the new index.html given in template-theme normal folder; All themes have identical index.html

**Comment by getmeapencil** [comment]
Replacing the code of new-themes with template-theme changes the positioning of box.
![Uploading original new theme.png…]()
![Uploading after using template.png…]()
Is this intended?

**Comment by Nainish-Rai** [comment]
@getmeapencil Yeah slightly changed position. Can you share screenshot If its changing much.

**Comment by Nainish-Rai** [comment]
@getmeapencil It is fine on 16:9 screen. You don't worry about it.
